src/StoreKeeper/WooCommerce/B2C/Commands/ProcessSingleTask.php

```diff
@@ -46,6 +48,14 @@ public function execute(array $arguments, array $assoc_arguments)
 
         // Add the removed tasks to the current task
         $task['meta_data']['removed_task_ids'] = $handler->getTrashedTasks();
+
+        $postExecutionMicroTime = microtime(true);
+        $postExecutionDateTime = date('Y-m-d H:i:s');
+        $executionDuration = $postExecutionMicroTime - $preExecutionMicroTime;
+
+        $task['meta_data']['pre_execution'] = $preExecutionDateTime;
+        $task['meta_data']['post_execution'] = $postExecutionDateTime;
+        $task['meta_data']['execution_duration_ms'] = $executionDuration;
```

Comment:
better to make columns for it, than you can sum up in sql

Reply:
Added field `execution_duration` field in `TaskModel`

Reply:
Retained `pre_execution` and `post_execution` datetime in `meta_data`.